artifactory_cleanup/artifactorycleanup.py

```diff
@@ -118,16 +116,29 @@ def main(self):
         table.align["Cleanup Policy"] = "l"
         total_size = 0
 
+        if is_running_under_teamcity():
```

Comment:
Can we move the block in a function?

Then we'll use the function here and below
```python
from artifactory_cleanup.context_managers import get_context_managers()
ctx_block, ctx_test = get_context_managers()
```

Reply:
Hi @allburov 

Thanks for the feedback, good point. I pushed a fix, but am leaving the comment unresolved so you can verify this fix properly address your comment